BookingsApi/BookingsApi.Contract/Requests/BulkJudiciaryPersonResponse.cs

```diff
@@ -0,0 +1,20 @@
+using System.Collections.Generic;
+
+namespace BookingsApi.Contract.Requests
+{
+    public class BulkJudiciaryPersonResponse
+    {
+        public BulkJudiciaryPersonResponse()
+        {
+            ErroredRequests = new List<JudiciaryPersonErrorResponse>();
+        }
+            
+        public List<JudiciaryPersonErrorResponse> ErroredRequests { get; set; }
+    }
+
+    public class JudiciaryPersonErrorResponse
```

Comment:
should this be in own file?

Reply:
I think its ok there.. unless you strongly feel otherwise

Reply:
i would in any other project but its quite common here at VH so i can let it fly :)